src/services/copyConfig/recaptcha/recaptchaConfiguration.js

```diff
@@ -0,0 +1,133 @@
+  }
+
+  async get() {
+    try {
+      const recaptchaResponse = await this.getRecaptcha().get(this.#site, this.#dataCenter)
+      if (recaptchaResponse.errorCode !== 0) {
+        throw new Error(`Error fetching reCAPTCHA policies: ${recaptchaResponse.errorMessage}`)
+      }
+
+      const policiesResponse = await this.#policy.get()
+      if (policiesResponse.errorCode !== 0) {
+        throw new Error(`Error fetching security and registration policies: ${policiesResponse.errorMessage}`)
+      }
+
+      const riskProvidersResponse = await this.#riskProviders.get(this.#site, this.#dataCenter)
+      if (riskProvidersResponse.errorCode !== 0) {
+        throw new Error(`Error fetching Risk Providers configuration: ${riskProvidersResponse.errorMessage}`)
+      }
+
+      return {
+        errorCode: recaptchaResponse.errorCode,
+        recaptchaConfig: recaptchaResponse.Config,
+        securityPolicies: policiesResponse.security,
+        registrationPolicies: policiesResponse.registration,
+        riskProvidersConfig: riskProvidersResponse.config,
+      }
+    } catch (error) {
+      throw new Error(`Error in RecaptchaConfiguration.get: ${error}`)
+    }
+  }
+
+  async setRecaptchaConfig(site, dataCenter, recaptchaConfig) {
+    const response = await this.getRecaptcha().set(site, dataCenter, recaptchaConfig)
+    if (response.errorCode === 0) {
+      return response
+    } else {
+      throw new Error(`Error setting reCAPTCHA configuration: ${response.errorMessage}`)
+    }
+  }
+
+  async setPolicies(targetSite, targetDataCenter, securityPolicies, registrationPolicies) {
+    try {
+      const newSecurityConfig = {
+        riskAssessmentWithReCaptchaV3: securityPolicies.riskAssessmentWithReCaptchaV3,
+        riskAssessmentWithTransUnion: securityPolicies.riskAssessmentWithTransUnion,
+      }
+
+      const newConfig = {
+        security: newSecurityConfig,
+        registration: {
+          requireCaptcha: registrationPolicies.requireCaptcha,
+        },
+      }
+
+      const response = await this.#policy.set(targetSite, newConfig, targetDataCenter)
+      if (response.errorCode === 0) {
+        return response
+      } else {
+        throw new Error(`Error fetching current policies: ${response.errorMessage}`)
+      }
+    } catch (error) {
+      throw new Error(`Error in setPolicies: ${error.message || error}`)
+    }
+  }
+
+  async setRiskProvidersConfig(site, dataCenter, riskProvidersConfig) {
+    const response = await this.#riskProviders.set(site, dataCenter, riskProvidersConfig)
+    if (response.errorCode === 0) {
+      return response
+    } else {
+      throw new Error(`Error setting Risk Providers configuration: ${response.errorMessage}`)
+    }
+  }
+
+  async copy(targetSite, targetDataCenter) {
+    const config = await this.get()
+    const dataCenter = targetDataCenter.dataCenter || targetDataCenter
```

Comment:
See the origin of the targetDataCenter.dataCenter or dataCenter, so we can remove this code waste.